chatbees/tests/regression_test.py

```diff
@@ -8,8 +9,8 @@
 import chatbees as cb
 from chatbees.server_models.doc_api import AnswerReference, ExtractType, ExtractedTable
 
-TEST_AID = os.environ.get('ENV_TEST_AID')
-TEST_APIKEY = os.environ.get('ENV_TEST_APIKEY')
+TEST_AID = 'BK5XN4QN'
```

Comment:
please revert this, and set env for test.
also better add a separate test file for local fs tests.